src/FxCop/System.Globalization.Analyzers/Test/Globalization/CA1309UseOrdinalStringComparisonTests.cs

```diff
@@ -6,14 +6,15 @@
 using Xunit;
 
 namespace System.Globalization.Analyzers.UnitTests
-{                              
+{
     public sealed class CA1309UseOrdinalStringComparisonTests : DiagnosticAnalyzerTestBase
     {
 #pragma warning disable CS0219
```

Comment:
Not sure. It was there in original template .cs file, so I just kept it.
It's removed now.